### PR TITLE
feat: add Post Card

### PR DESCRIPTION
**Pull request by sauldeleon**
Created a PostCard component for Storybook

image

### CODE REVIEW

File: packages/blog/src/components/home/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { HeaderImage } from '../HeaderImage'
+
+export function HomePage() {
+  return (
+    <HeaderImage
+      imgSrc="https://images.unsplash.com/photo-1506477331477-33d5d8b3dc85?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=868&q=80"
+      date={new Date()}
+      categories={['travel', 'lifestyle']}
```
Comment (paulaguijarro) [inline packages/blog/src/components/home/HomePage/HomePage.tsx:8]: Why are the categories in English and the title and subtitle in Spanish?

File: packages/common/src/components/PostCard/PostCard.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  right: 0;
+  margin-left: auto;
+  margin-right: auto;
+`
+
+const Template: Story<PostCardProps> = args => <PostCard {...args} />
+
+export const Default = Template.bind({})
+Default.args = {
+  categories: ['Travel'],
```
Comment (paulaguijarro) [inline packages/common/src/components/PostCard/PostCard.stories.tsx:37]: categories are going to be capitalized? I think they were in lowercase elsewhere.

File: packages/common/src/components/PostCard/PostCard.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+MultipleCategories.args = {
+  categories: [
+    '1 Travel',
+    '2 Lifestyle',
+    '3 Lifestyle',
+    '8 Paranormal activity',
+    '4 Lifestyle',
+    '6 Lifestyle',
+    '7 Lifestyle',
+    '8 Paranormal activity',
```
Comment (paulaguijarro) [inline packages/common/src/components/PostCard/PostCard.stories.tsx:56]: Paranormal

**Review by paulaguijarro — APPROVED** [review]

File: packages/blog/src/components/home/HomePage/HomePage.tsx (hunk 1 above)
Comment (sauldeleon) [reply packages/blog/src/components/home/HomePage/HomePage.tsx:8]: Theese are all mocked data. It will be changed later with proper data

**Pull request merged**